lib/Pod/Simple/HTML.pm

```diff
@@ -797,7 +797,7 @@ sub resolve_man_page_link {
   $section ||= 1;
 
   return $self->man_url_prefix . "$section/"
-      . $self->manpage_url_escape($page)
+      . $self->manpage_url_escape($page) . ".$section"
```

Comment:
This breaks existing code that sets `man_url_prefix` or `man_url_postfix` because now there is an additional hardcoded `.$section`  string in the middle.

Also, the comment on `man_url_postfix` says it defaults to `""`, which is no longer true.

Reply:
The changelog suggests overriding the method by subclassing for URL changes:
```
Turned man page links into links to man.he.net in Pod::Simple::HTML
    and Pod::Simple::XHTML output. Subclass and override
    resolve_man_page_link() to use a different URL.
```

Fixed the comment.